helm_image_updater/config.py

```diff
@@ -24,6 +24,7 @@
 
 # Constants
 DEV_STACKS = ["dev-keboola-gcp-us-central1"]
+EXCLUDED_STACKS = ["dev-keboola-gcp-us-east1-e2e"]
```

Comment:
pridavam podporu pre excluded stacky a zatial tam davam e2e stack. To znamena ze tieto stacky nebudu sucastou automatickeho vyberu stackov ale stale ich bude mozne nasadit cez override-stack